packages/expo-notifications/src/registerTaskAsync.ts

```diff
@@ -3,11 +3,16 @@ import { UnavailabilityError } from 'expo-modules-core';
 import BackgroundNotificationTasksModule from './BackgroundNotificationTasksModule';
 
 /**
- * When a notification is received while the app is backgrounded, using this function you can set a callback that will be run in response to that notification.
- * Under the hood, this function is run using `expo-task-manager`. You **must** define the task first, with [`TaskManager.defineTask`](./task-manager#taskmanagerdefinetasktaskname-taskexecutor).
- * Make sure you define it in the global scope. It is recommended to define only one task because running a task can have a high cost.
+ * Call `registerTaskAsync` to set a callback (task) that will run in response to when a notification is received while the app is in foreground, background or terminated.
```

Comment:
```suggestion
 * Call `registerTaskAsync` to set a callback (task) that will run in response to when a notification is received while the app is in foreground, background, or terminated.
```

Missing oxford comma here because we are using three items.